docs/quickstarts/javascript/react/conditional-ui-rendering.mdx

```diff
@@ -220,6 +203,17 @@ if (asertoError) {
 }
 ```
 
+Next, we’ll add a to call `getDisplayState`:
+
+```javascript
+const displayStateMap =
+  loading || asertoError
+    ? { visible: false, enabled: false }
+    : getDisplayState("GET", "/api/protected");
+```
+
+In case the SDK is not yet loaded or returns an error, the `disaplyStateMap` variable will be set to `{ visible: false, enabled: false }`. If the SDK completed loading properly, it'll issue a call to the Display State Map endpoint that is set up on our server, and retrieve the `visible` and `enabled` decisions from the authorizer for our policy. Note that we specify the `GET` method and the `/api/protected` path in our call to `getDisplayState` since there could be multiple policies we may to get the display state for - this call will specifically check the policy `asertodemo.GET.api.protected`.
```

Comment:
this isn't actually true. the getDisplayState will just look at the internal state retrieved from `init` (or `reload`), and navigate the object (first into the HTTP method, then into the path), and return the specific { visible, allowed } object for that method and route.  So, no network call. Hence there is no "await" in front of the "getDisplayState".